src/shared/components/annotation/OncoKB.tsx

```diff
@@ -24,6 +24,7 @@ export interface IOncoKbProps {
     evidenceCache?: OncoKbEvidenceCache;
     evidenceQuery?: Query;
     pubMedCache?: OncokbPubMedCache;
+    geneNotExist:boolean;
```

Comment:
geneNotAnnotated?

Reply:
geneExist is the keyword used in the oncokb API, so I used exist for coherence